soh/src/code/audio_load.c

```diff
@@ -1366,16 +1368,23 @@ void AudioLoad_Init(void* heap, size_t heapSize) {
     int startingSeqNum = MAX_AUTHENTIC_SEQID; // 109 is the highest vanilla sequence
     qsort(customSeqList, customSeqListSize, sizeof(char*), strcmp_sort);
 
+    int seqNum = startingSeqNum;
+
     for (size_t i = startingSeqNum; i < startingSeqNum + customSeqListSize; i++) {
+        // ensure that what would be the next sequence number is actually unassigned in AudioCollection
+        while (AudioCollection_GetSequenceName(seqNum) != NULL) {
+            seqNum++;
+        }
```

Comment:
it'd be good to add some clarifying comments explaining *why* we need to check this here, and why it's a better solution than just setting `MAX_AUTHENTIC_SEQID` to be a bigger number

Reply:
Is the fact that there are currently like 20 unused seqNums between the end of the end of the vanilla sequences and the sound fonts at 130-136 causing the main issue enough?

Reply:
i think the bigger thing is what you mentioned before about some sequences
> in the low 2000, mid 6000, and 10-12k ranges, too

if it was just 20 unused ones we would probably be fine just setting the max to after the sound fonts

Reply:
Oh, OK. I'll get some specifics about what those others are and add them to the explanation.

Reply:
Oh, and I was wrong, they're instruments there, not sound fonts. Those are later I think XD